compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -364,6 +376,19 @@ pub fn expand_test_or_bench(
     }
 }
 
+fn get_location_info(cx: &ExtCtxt<'_>, item: &ast::Item) -> (Symbol, usize, usize, usize, usize) {
+    let span = item.ident.span;
+    let (source_file, lo_line, lo_col, hi_line, hi_col) =
+        cx.sess.source_map().span_to_location_info(span);
+
+    let file_name = match source_file {
+        Some(sf) => sf.name.display(FileNameDisplayPreference::Remapped).to_string(),
```

Comment:
@oli-obk the fix was to change ::Local to ::Remapped and then add ```--remap-path-prefix={{src-base}}=/the/src``` in tests-are-sorted.rs